### PR TITLE
Label output for TCs

### PR DESCRIPTION
**Pull request by adcroft**
- Resizes tc3 to 10x8 to be same size as other TCs
- Labels stdout from each TC and test, e.g.
```
tc2.dim.t: MOM Date      1/01/02 00:00:00     24: En 2.773492E-04, MaxCFL  0.00014, Mass 1.396373904329E+20, Salt  35.00000000000, Temp  13.34855257895
tc2.dim.t:     Total Energy: 436132E673084A4D  3.8728319662969448E+16
tc2.dim.t:     Total Mass:   1.3963739043293897E+20, Change:  -2.3821105957031250E+02 Error: -2.37768E+02 (-1.7E-18)
tc2.dim.t:     Total Salt:   4.8873086651528632E+18, Change:  -2.7443200000000002E+02 Error: -2.74432E+02 (-5.6E-17)
tc2.dim.t:     Total Heat:   7.3160314141103079E+24, Change:   7.6890406992664855E+19 Error:  1.31315E+08 ( 1.8E-17)
```
See https://travis-ci.org/adcroft/MOM6/jobs/587225628 for complete example

### CODE REVIEW

File: .testing/Makefile [hunk 1]
```diff
@@ -218,13 +218,20 @@ else
 MPIRUN_CMD=$(MPIRUN) $(if $(1),-x $(1),)
 endif
 
+# Rule to build <tc>/ocean.stats.<tag>
+# $(1) - <tag>
+# $(2) - <executable-label>
+# $(3) - report to codecov
+# $(4) - Parameter line for MOM_override
+# $(5) - Environment for MPIRUN
+# $(6) - Number of PEs
```
Comment (marshallward) [inline .testing/Makefile]: I had these defined up in line 205, did you also want to remove the other one?

Comment (adcroft) [reply .testing/Makefile]: Ha! I didn't see those comments. It doesn't seem right to have an `ifeq` with it's own comments in between the description and function. I'll delete mine and move yours down?

Comment (marshallward) [reply .testing/Makefile]: Yeah I think that works.  You can just delete mine if it's easier (or merge them into a grand unified comment)

Comment (adcroft) [reply .testing/Makefile]: I rearranged and combined. PR has been updated.

**Comment by marshallward** [comment]
No changes to code so no need for regression tests.

**Pull request merged**